service/extensions/extensions.go

```diff
@@ -128,18 +128,15 @@ type Settings struct {
 
 	// Factories maps extension type names in the config to the respective component.ExtensionFactory.
 	Factories map[config.Type]component.ExtensionFactory
-
-	// ServiceExtensions are the ordered list of extensions configured for the service.
-	ServiceExtensions []config.ComponentID
 }
 
-// Build builds Extensions from config.
-func Build(ctx context.Context, set Settings) (*Extensions, error) {
+// New creates a new Extensions from Config.
+func New(ctx context.Context, set Settings, cfg Config) (*Extensions, error) {
```

Comment:
I like the idea of using `Config` like other components do, but what gets put in `Config` vs `Settings`?

Reply:
That distinction makes sense, thanks for clarifying.